runtime-native/build.gradle

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+plugins {
+    id 'java'
+}
+
+description = 'Ballerina - Runtime Java Utils'
+
+dependencies {
+    compile group: 'org.ballerinalang', name: 'ballerina-runtime', version: "${ballerinaLangVersion}"
```

Comment:
Do we need runtime as an external dependency here, in runtime stdlib?

Reply:
Yes, stdlib runtime module is an API for exposing Ballerina runtime related functions. So this module uses functions of the actual Ballerina language runtime. This dependency is not the stdlib runtime. It is the actual Ballerina language runtime.